src/sources/exec/mod.rs

```diff
@@ -480,6 +484,64 @@ fn handle_exit_status(config: &ExecConfig, exit_status: Option<i32>, exec_durati
     });
 }
 
+#[cfg(unix)]
+fn shutdown_child(
+    pid: Option<u32>,
+    _child: &tokio::process::Child,
```

Comment:
Since `pid` is just `child.id()` above, and we are already passing in a reference to `child`, do we need the extra parameter here?

Reply:
Good call 👍